### ISSUE TITLE
Fix call to get main use of each building from OSM

### ISSUE THREAD

**Issue by martin-mosteiro** [architecture-building-systems/CityEnergyAnalyst#2947]
**Describe the bug**
When running the CEA zone helper, all buildings get imported as *MULTI_RES*. I thought using the `category` field obtained from OSM could be used in order to get the main use in the building... then noticed there's already code to do so in CEA! However, even though by default buildings should be assigned the right typology based on OSM, they are not.

**To Reproduce**
1. Create a new scenario in the CEA dashboard
2. Select any zone you want to – as long as not _all_ buildings are residential
3. Check the typology – see that non-residential buildings are also labeled as *MULTI_RES*

**Solution**
Fixing this issue was a super easy fix – the wrong column name was used in the creation of the typology file, so the categories from OSM were never _actually_ used in creating the typology file.

However, there's relatively few osm categories that are hard-coded into the list categories processed by CEA:
```
    if occupancy_type == "Get it from open street maps":
        no_buildings = typology_df.shape[0]
        for index in range(no_buildings):
            if zone_df.loc[index, "category"] == "yes":
                typology_df.loc[index, '1ST_USE'] = "MULTI_RES"
                typology_df.loc[index, "REFERENCE"] = "CEA - assumption"
            elif zone_df.loc[index, "category"] == "residential" or zone_df.loc[index, "category"] == "apartments":
                typology_df.loc[index, '1ST_USE'] = "MULTI_RES"
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            elif zone_df.loc[index, "category"] == "commercial" or zone_df.loc[index, "category"] == "civic":
                typology_df.loc[index, '1ST_USE'] = "OFFICE"
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            elif zone_df.loc[index, "category"] == "school":
                typology_df.loc[index, '1ST_USE'] = "SCHOOL"
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            elif zone_df.loc[index, "category"] == "garage" or zone_df.loc[index, "category"] == "garages" or \
                    zone_df.loc[index, "category"] == "warehouse":
                typology_df.loc[index, '1ST_USE'] = "PARKING"
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            elif zone_df.loc[index, "category"] == "house" or zone_df.loc[index, "category"] == "terrace" or \
                    zone_df.loc[index, "category"] == "detached":
                typology_df.loc[index, '1ST_USE'] = "SINGLE_RES"
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            elif zone_df.loc[index, "category"] == "retail":
                typology_df.loc[index, '1ST_USE'] = "RETAIL"
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            elif zone_df.loc[index, "category"] == "industrial":
                typology_df.loc[index, '1ST_USE'] = "INDUSTRIAL"
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            elif zone_df.loc[index, "category"] == "warehouse":
                typology_df.loc[index, '1ST_USE'] = "INDUSTRIAL"
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            else:
                typology_df.loc[index, '1ST_USE'] = "MULTI_RES"
                typology_df.loc[index, "REFERENCE"] = "CEA - assumption"
```
Maybe a longer list could be provided and stored as a dict in `constants.py`? That way (a) CEA would process more building categories by default, and (b) it would be way easier to add further categories into list of CEA categories can process without adding further hard-coded `elif` statements. The code could look something like this:
```
    if occupancy_type == "Get it from open street maps":
        no_buildings = typology_df.shape[0]
        for index in range(no_buildings):
            if zone_df.loc[index, "category"] in OPEN_STREET_MAP_CATEGORIES.keys():
                typology_df.loc[index, '1ST_USE'] = OPEN_STREET_MAP_CATEGORIES[zone_df.loc[index, "category"]]
                typology_df.loc[index, "REFERENCE"] = "OSM - as it is"
            else:
                typology_df.loc[index, '1ST_USE'] = "MULTI_RES"
                typology_df.loc[index, "REFERENCE"] = "CEA - assumption"
```
where
```
OPEN_STREET_MAP_CATEGORIES = {'residential': 'MULTI_RES', 'apartments': 'MULTI_RES', 
                                                             'commercial': 'OFFICE', 'civic': 'OFFICE',
                                                             'school': 'SCHOOL',
                                                             [...]
                                                            }
```

**Comment by martin-mosteiro** [comment]
Fixed the issue by changing the column in typology_df where the main use types were assigned from ~'USE_A'~ to '1ST_TYPE' to be consistent with the output columns.

Then I enhanced this by adding additional categories ([from the OpenStreetMap wiki](https://wiki.openstreetmap.org/wiki/Key:building)) and moving these categories to a constant to make it easier for further categories to be added down the line.

Unfortunately, I do not have access to a computer with a usable CEA installation – nor will I have access to one any time soon. So it'd be great if someone could test that the fix works. Analogously to the steps "To Reproduce" from the issue description, this can be tested as follows:

**To Test**
Steps to verify the behavior doesn't happen any more:
1. Create a new scenario in the CEA dashboard
2. Select any zone you want to – as long as not all buildings are residential
3. Check the typology – make sure that non-residential buildings are not labeled as **MULTI_RES**.
NOTE: "yes" buildings should still be assigned as "MULTI_RES", just listed as "CEA - assumption".
